assets/js/components/Dashboard/PaMessageForm/MainForm.tsx

```diff
@@ -502,53 +511,91 @@ const NewPaMessageHeader = ({
     );
   };
 
-  return associatedAlert ? (
-    <Row md="auto" className="align-items-center">
-      <div className="associated-alert-header">
-        Associated Alert: Alert ID{" "}
-        {typeof associatedAlert === "string"
-          ? associatedAlert
-          : associatedAlert.id}
-        <Button
-          variant="link"
-          onClick={() => {
-            onClearAssociatedAlert();
-          }}
-        >
-          Clear
-        </Button>
-        {typeof associatedAlert === "string" ? (
-          <div className="alert-ended">
-            Alert has ended and is no longer available.
-          </div>
-        ) : (
-          formatActivePeriod(associatedAlert.active_period)
-        )}
-      </div>
-    </Row>
-  ) : (
-    <>
-      <Row md="auto" className="align-items-center unassociated-alert-header">
-        <Button
-          variant="link"
-          className="pr-0 associate-alert-button"
-          onClick={() => {
-            setEndWithEffectPeriod(true);
-            navigateTo(Page.ALERTS);
-          }}
-        >
-          Associate with alert
-        </Button>
-        (Optional)
+  const formatMessageType = (messageType: MessageType) => {
+    switch (messageType) {
+      case "psa":
+        return "PSA";
+      case "emergency":
+        return "Emergency";
+      default:
+        return "";
+    }
+  };
+
+  let content;
```

Comment:
nit: Since we're just returning the content immediately, we could put `return` statements inside each `if` and avoid a `let` here.